en/simulation/hitl.md

```diff
@@ -13,6 +13,7 @@ Airframe | `SYS_AUTOSTART` | X-Plane | Gazebo | jMAVSim
 ---|---|---
 <a href="../airframes/airframe_reference.md#plane_simulation_(plane)_hilstar_(xplane)">HILStar (X-Plane)</a> | 1000 | Y | |
 <a href="../airframes/airframe_reference.md#copter_simulation_(copter)_hil_quadcopter_x">HIL Quadcopter X</a> | 1001 | | Y | Y
+<a href="../airframes/airframe_reference.md#vtol_simulation_standard_vtol_(gazebo)">Standard VTOL</a> | 1002 | | Y |
```

Comment:
@hamishwillee Not yet sure if this will be the correct link, that section will be autogenerated as soon as the pull request on the firmware side will be merged.

Reply:
Cheers. I could work it out, but easiest just to wait and see.